### PR TITLE
chore: 🤖 update config in prep of testnet and add testnet doc

### PR DESCRIPTION
**Pull request by cosmic-vagabond**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -1,64 +1,54 @@
...
-  - name: alice
+  - name: treasury
     coins:
-      - 200000000000000000000aelys
-  - name: bob
+      - "9000000000000000uelys"
+  - name: seed
     coins:
-      - 100000000000000000000aelys
+      - "9000000000000000uelys"
```
Comment (jelysn) [inline config.yml:8]: elys decimal changed from 18 to 6?

File: readme.md [hunk 2]
```diff
@@ -543,3 +575,22 @@ To query the list of the time-based-inflation entries in the tokenomics module,
...
+```
+git tag v0.1
+git push origin v0.1
+```
+
+After a draft release is created, make your final changes from the release page and publish it.
+
+## Learn more
+
+- [Ignite CLI](https://ignite.com/cli)
```
Comment (jelysn) [inline readme.md:592]: Do we need to put Ignite CLI document on elys repo?

**Comment by jelysn** [comment]
I think README.md should include more information about Elys network and adding references to other documents and the development notes like ignite release or etc, could be put on `dev_notes.md` or somewhere.

File: readme.md [hunk 3]
```diff
@@ -543,3 +575,22 @@ To query the list of the time-based-inflation entries in the tokenomics module,
 ```
 elysd q tokenomics list-time-based-inflation
 ```
+
+## Release
+
+To release a new version of your blockchain, create and push a new tag with `v` prefix. A new draft release with the configured targets will be created.
```
[the remark this answers is not in the document]
Comment (cosmic-vagabond) [reply readme.md:581]: good call out

File: readme.md (hunk 2 above)
Comment (cosmic-vagabond) [reply readme.md:592]: I want to keep it here until we add our own Makefile

**Comment by cosmic-vagabond** [comment]
> I think README.md should include more information about Elys network and adding references to other documents and the development notes like ignite release or etc, could be put on `dev_notes.md` or somewhere.

agree it's coming

**Review by AustinoBombino — APPROVED** [review]

**Pull request merged**